watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/supercritical_sludge_to_gas/supercritical_sludge_to_gas.py

```diff
@@ -0,0 +1,312 @@
+    costing_kwargs = {"default": {"flowsheet_costing_block": m.fs.costing}}
+    m.fs.ATHTL.costing = UnitModelCostingBlock(**costing_kwargs)
+    m.fs.salt_precipitation.costing = UnitModelCostingBlock(**costing_kwargs)
+    m.fs.HTG.costing = UnitModelCostingBlock(**costing_kwargs)
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_electricity_intensity(m.fs.product_H2O.properties[0].flow_vol)
+    m.fs.costing.add_LCOW(m.fs.product_H2O.properties[0].flow_vol)
+
+    m.fs.costing.LCOG = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+        )
+        / (
+            m.fs.costing.utilization_factor
+            * pyunits.convert(
+                m.fs.product_natural_gas.properties[0].flow_mass_comp["carbon_dioxide"],
+                to_units=pyunits.kg / m.fs.costing.base_period,
+            )
+        ),
+        doc="Levelized cost of CO2 for production of renewable natural gas",
+    )
+
+    m.fs.costing.LCOC = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+        )
+        / (
+            m.fs.costing.utilization_factor
+            * pyunits.convert(
+                m.fs.product_CO2.properties[0].flow_mass_comp["carbon_dioxide"],
+                to_units=pyunits.kg / m.fs.costing.base_period,
+            )
+        ),
+        doc="Levelized cost of CO2 from AT-HTL",
+    )
+
+    m.fs.costing.LCOS = Expression(
+        expr=(
+            m.fs.costing.total_capital_cost * m.fs.costing.capital_recovery_factor
+            + m.fs.costing.total_operating_cost
+        )
+        / (
+            m.fs.costing.utilization_factor
+            * pyunits.convert(
+                (
+                    m.fs.product_salts.properties[0].flow_mass_comp["organic_solid"]
+                    + m.fs.product_salts.properties[0].flow_mass_comp["inorganic_solid"]
+                ),
+                to_units=pyunits.kg / m.fs.costing.base_period,
+            )
+        ),
+        doc="Levelized cost of Sulfates, Nitrates and Phosphates",
+    )
+
+
+def initialize_costing(m):
+    m.fs.costing.initialize()
+
+
+def display_costing(m):
+    m.fs.costing.total_capital_cost.display()
+    m.fs.costing.total_operating_cost.display()
+    m.fs.costing.LCOW.display()
+
+    print("\nUnit Capital Costs\n")
+    for u in m.fs.costing._registered_unit_costing:
+        print(
+            u.name,
+            " :   ",
+            value(pyunits.convert(u.capital_cost, to_units=pyunits.USD_2018)),
+        )
+
+    print("\nUtility Costs\n")
+    for f in m.fs.costing.flow_types:
+        print(
+            f,
+            " :   ",
+            value(
+                pyunits.convert(
+                    m.fs.costing.aggregate_flow_costs[f],
+                    to_units=pyunits.USD_2018 / pyunits.year,
```

Comment:
Comment on a few recent PRs suggested using `pyunits.USD_2020` for the AMO projects.

Reply:
Thank you, it's a good point. I've modified to `USD_2020`.